docs/ManualDeploymentGuideScalarDLonAWS.md

```diff
@@ -0,0 +1,292 @@
+
+### Recommendations
+
+* Kubernetes should be private in production and should provide secure access with SSH or VPN. You can use a bastion server as a host machine for secure access to the private network.
+* Subnets should be in multiple availability zones, to enable fault tolerance for the production use.
+* Kubernetes cluster should be public to enable the envoy public endpoint. The Kubernetes cluster in public is not recommended for production.
+
+> **TIP** 
+>
+> If you plan to create three or more nodes in a Kubernetes node group for high availability, create at least 3 subnets in different AZs for the Kubernetes cluster.
+
+[Create an Amazon VPC](https://docs.aws.amazon.com/directoryservice/latest/admin-guide/gsg_create_vpc.html) with the above requirements and recommendations along with your organizational or application standards.
+
+## Step 2. Set up a Scalar DL supported database 
+
+Scalar DL can run on various databases, which is an essential component for underlying storage. This section will help you to set up a Scalar DL supported database.
+
+### Requirements
+
+* You must have a Scalar DL supported managed database.
+
+Follow the [Set up a Scalar DL supported database](ScalarDLSupportedDatabase.md) document.
+
+## Step 3. Configure EKS
+
+Amazon EKS automatically manages the availability and scalability of the Kubernetes control plane nodes, so it will help you to achieve the high availability and scalability of Scalar DL. 
+This section will help you to configure a Kubernetes service for Scalar DL deployment.
+
+### Prerequisites
+Install the following tools on your host machine:
+* [awscli](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2-linux.html#cliv2-linux-install): In this tutorial, awscli is used to create a kubeconfig file to access the EKS cluster.
+* [kubectl](https://kubernetes.io/docs/tasks/tools/): Kubernetes command-line tool to manage EKS cluster. Kubectl 1.19 or higher is required.
+
+### Requirements
+
+* You must have an EKS cluster with Kubernetes version 1.19 or above in order to use our most up-to-date configuration files.
+* Kubernetes node group must be labeled with key as `agentpool` and value as `scalardlpool` for ledger and envoy deployment.
+
+### Recommendations
+
+* Kubernetes node size should be `m5.large` for deploying `ledger` and `envoy` pods, because each node requires 2 vcpu and 8gb ram for deploying ledger and envoy pods.
+* Node groups should have at least 3 nodes for high availability in production.
+
+
+### Create a Kubernetes cluster	
+
+Kubernetes cluster is a primary component of EKS. Scalar DL requires a single EKS cluster for deploying ledger, envoy and monitor services. This section will help you to create an EKS cluster. 
+
+[Create an Amazon EKS cluster](https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html) with the above requirements.
+
+> **Tip**
```

Comment:
Ditto

Reply:
Removed Quotes and changed tip to Italics.